sklearn/kernel_ridge.py

```diff
@@ -36,6 +36,8 @@ class KernelRidge(MultiOutputMixin, RegressorMixin, BaseEstimator):
 
     Read more in the :ref:`User Guide <kernel_ridge>`.
 
+	.. versionchanged:: 0.16
```

Comment:
I think it only makes sense to report the `versionadded` as estimator behavior change rather frequently...

Reply:
Changes in parameter semantics are worth noting... But they need detail